packages/paste-website/src/pages/utilities/grid/index.mdx

```diff
@@ -0,0 +1,520 @@
+
+<Callout variant="secondary">
+  <CalloutText>For spacing content in a layout wider than 1440 pixels, we recommend using a 24-pixel gutter. Use $space-40 to get a 24-pixel gutter.</CalloutText>
+</Callout>
+
+<Callout variant="secondary">
+  <CalloutText>For spacing content in a layout narrower than 1440 pixels, we recommend using a 16-pixel gutter. Use $space-30 to get a 16-pixel gutter.</CalloutText>
+</Callout>
+
+#### Basic page layout
+
+Let's look at a common use case where we need to layout a page with a sidebar and a grid within the page content area.
+
+<LivePreview scope={{Grid, Column, Box, Card, Heading, Paragraph, Text}} language="jsx">
+  {`<Card padding="space70">
+  <Grid gutter="space30">
+    <Column span={3}>
+      <Box backgroundColor="colorBackgroundPrimaryDarker" padding="space70" borderRadius="borderRadius20">
+        <Text as="h4" textColor="colorTextInverse">Sidebar Header</Text>
+        <Text textColor="colorTextInverse">Chillwave fixie sustainable helvetica etsy. Prism whatever ethical.</Text>
+      </Box>
+    </Column>
+    <Column span={9}>
+      <Box marginBottom="space50">
+        <Card>
+          <Paragraph>Chillwave fixie sustainable helvetica etsy.</Paragraph>
+        </Card>
+      </Box>
+      <Grid gutter="space30">
+        <Column>
+          <Card>
+            <Paragraph>Chillwave fixie sustainable helvetica etsy.</Paragraph>
+          </Card>
+        </Column>
+        <Column>
+          <Card>
+            <Paragraph>Chillwave fixie sustainable helvetica etsy.</Paragraph>
+          </Card>
+        </Column>
+      </Grid>
+    </Column>
+  </Grid>
+</Card>`}
+</LivePreview>
+
+#### Responsive page layout
+
+Grid can also be used for responsive layouts.
+
+ - You can determine what size columns should be based on the breakpoints set on a theme. [See the Storybook example](https://twilio-labs.github.io/paste/?path=/story/utilities-grid--grid-2-column-responsive)
+ - Grid columns can also stack based on theme breakpoints. [See the Storybook example](https://twilio-labs.github.io/paste/?path=/story/utilities-grid--grid-2-column-responsive-stack)
+
+#### Component layout
+
+Grids can also be used for layouts within other components. For example, we can set up our card content to include 2 columns.
+
+<LivePreview scope={{Grid, Column, Card, Heading, Paragraph, TwilioIcon, TWILIO_RED}} language="jsx">
+  {`<Card padding="space70">
+  <Grid gutter="space30">
+    <Column span={2}>
+      <TwilioIcon color={TWILIO_RED} display="block" size={108} />
+    </Column>
+    <Column span={10}>
+      <Heading as="h4" variant="heading40">Card Heading</Heading>
+      <Paragraph>Chillwave fixie sustainable helvetica etsy.</Paragraph>
+    </Column>
+  </Grid>
+</Card>`}
+</LivePreview>
+
+### When to use Grid
+
+Use Grid whenever you need to layout or align content on a page or within a component. Grid should also be used for responsive page layouts.
```

Comment:
I don't know where or how, but I kind of feel like this might be a good place to say something like: "Since the grid is sized to the full width of its parent container, you may not want to use it for fixed width columns in a variable width container.